packages/admin/admin-rte/src/core/Rte.tsx

```diff
@@ -356,7 +392,7 @@ declare module "@mui/material/styles" {
 
     interface Components {
         CometAdminRte?: {
-            defaultProps?: ComponentsPropsList["CometAdminRte"];
+            defaultProps?: Partial<ComponentsPropsList["CometAdminRte"]>;
```

Comment:
This is already defined as `Partial` in `ComponentsPropsList`

```suggestion
            defaultProps?: ComponentsPropsList["CometAdminRte"];
```